metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/WaitForBuildIndicesStep.java

```diff
@@ -8,7 +8,7 @@
 @RequiredArgsConstructor
 public class WaitForBuildIndicesStep implements BootstrapStep {
```

Comment:
Minor: Make this an UpgradeStep if you want it to be written to MySQL after the first execution (it will also be skipped thereafter without needing to check kafka at all)